libs/eslint-plugin-vx/src/rules/gts_func_style.ts

```diff
@@ -38,7 +36,7 @@ const rule: TSESLint.RuleModule<'useFunctionDeclaration', readonly unknown[]> =
 
       return {
         VariableDeclarator(node: TSESTree.VariableDeclarator): void {
-          const declaration = node.parent as TSESTree.VariableDeclaration;
+          const declaration = node.parent;
```

Comment:
The types of various node parents was refined in https://github.com/typescript-eslint/typescript-eslint/pull/9560.